packages/firebase_performance/lib/src/firebase_performance.dart

```diff
@@ -17,7 +15,7 @@ class FirebasePerformance {
   FirebasePerformance._(this._handle) {
     channel.invokeMethod<bool>(
       'FirebasePerformance#instance',
-      <String, dynamic>{'handle': _handle},
+      <String, Object>{'handle': _handle},
```

Comment:
<String, Object?>

same for all the other maps in the PR

Reply:
Adjusted them :)